### PR TITLE
Adding inital typography tokens

### PR DESCRIPTION
**Pull request by georgewrmarshall**
## Description
Adding Typography tokens for JS tokens for mobile and CSS for web apps. This also updates the folder structure of the library and places all CSS in JS in a `/js/` folder. Open to discussion about this.

Fixes: https://github.com/MetaMask/design-tokens/issues/65

### CODE REVIEW

File: docs/BrandColors.mdx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { Meta, Canvas, Story } from '@storybook/addon-docs/blocks';
+import { Meta, Canvas, Story } from '@storybook/addon-docs';
 import LinkTo from '@storybook/addon-links/react';
 
 <Meta title="Design Tokens/Brand Colors" />
```
Comment (georgewrmarshall) [inline docs/BrandColors.mdx:4]: Fixing deprecated import

Screen Shot 2022-04-25 at 8 27 39 PM

File: docs/ThemeColors.mdx [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { Meta, Canvas, Story } from '@storybook/addon-docs/blocks';
+import { Meta, Canvas, Story } from '@storybook/addon-docs';
 
 import tokens from '../src/figma/tokens.json';
```
Comment (georgewrmarshall) [inline docs/ThemeColors.mdx:4]: Same as above

File: docs/components/Text/Text.tsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  Omit<React.ComponentPropsWithoutRef<C>, keyof Props<C>>;
+
+export const Text = <C extends React.ElementType = 'span'>({
+  style,
+  children,
+  as,
+}: TextProps<C>) => {
+  const Component = as || 'span';
+  return <Component style={style}>{children}</Component>;
+};
```
Comment (georgewrmarshall) [inline docs/components/Text/Text.tsx:28]: Might need some typescript help with this component @Cal-L

File: src/js/typography/fontSizes.ts [hunk 4]
```diff
@@ -0,0 +1,11 @@
+export const fontSizes = {
+  fontSize: 0,
```
[the remark this answers is not in the document]
Comment (georgewrmarshall) [reply src/js/typography/fontSizes.ts:2]: Nice catch. I think my train of thought was that we should have 0 values for everything but can't remember the reason. Was dumb will remove 😅

[the remark this answers is not in the document]
Comment (georgewrmarshall) [reply src/js/typography/fontSizes.ts]: Removed in https://github.com/MetaMask/design-tokens/pull/94/commits/e78e1719f449938394499ee1205996219ed424e0

**Review by Cal-L — APPROVED** [review]
LGTM

**Review by Cal-L — APPROVED** [review]
LGTM

**Pull request merged**